src/GithubClientFactory.ts

```diff
@@ -0,0 +1,41 @@
+import {GithubClientConfigs} from './Config';
+import {Logger} from "./Logger";
+/* const { throttlingPlugin } = require("@octokit/plugin-throttling");
+const { retryPlugin } = require("@octokit/plugin-retry");
+
+ */
+const Octokit = require("@octokit/rest").plugin(require("@octokit/plugin-throttling")).plugin(require("@octokit/plugin-retry"));
```

Comment:
Le typage d'Octokit est déjà dans la lib originale, on va en profiter! `require` perd le typage, on va favoriser `import` à la place.

```suggestion
import { Octokit } from "@octokit/rest";
import { OctokitThrottling } from "@octokit/plugin-throttling";
import { OctokitRetry } from "@octokit/plugin-retry";

const MyOctokit = Octokit.plugin(OctokitThrottling).plugin(OctokitRetry);
```

Reply:
J'ai essayé et ça ne fonctionne pas. Il y a un fuck dans Octokit qui empêche de fonctionner comme ça.